lib/dialects/postgres/query-generator.js

```diff
@@ -476,6 +476,12 @@ class PostgresQueryGenerator extends AbstractQueryGenerator {
 
         if (attribute.type instanceof DataTypes.ARRAY) {
           type += '[]';
+
+          // Add table and field name if not exists on attributes with ARRAY(ENUM) type
+          if (!attribute.fieldName) {
+            attribute.fieldName = options.key;
+            attribute.tableName = options.table.tableName || options.table;
```

Comment:
`attribute.tableName` is not used anywhere else, I think.

Reply:
Yes this looked a bit weird to me as well ... but i had to pass table info at
https://github.com/sequelize/sequelize/pull/11517/files#diff-7e347c4738c97261969fbeb802a3c363R511